robot/web/web.go

```diff
@@ -520,10 +521,10 @@ func (svc *webService) runWeb(ctx context.Context, options weboptions.Options) (
 		return err
 	}
 
-	if err := svc.refreshResources(); err != nil {
+	if err := svc.initAPIResourceCollections(ctx, false); err != nil {
```

Comment:
I'm just going to ignore the fact I have no idea what this is doing!

Reply:
it doesn't do anything fancy, just initializes all the api service collections and registers them to the server